src/connector/src/source/kafka/enumerator.rs

```diff
@@ -227,17 +227,6 @@ async fn build_kafka_client(
     let client_ctx = RwConsumerContext::new(ctx_common);
     let client: KafkaConsumer = config.create_with_context(client_ctx).await?;
 
-    // Note that before any SASL/OAUTHBEARER broker connection can succeed the application must call
-    // rd_kafka_oauthbearer_set_token() once – either directly or, more typically, by invoking either
-    // rd_kafka_poll(), rd_kafka_consumer_poll(), rd_kafka_queue_poll(), etc, in order to cause retrieval
-    // of an initial token to occur.
-    // https://docs.confluent.io/platform/current/clients/librdkafka/html/rdkafka_8h.html#a988395722598f63396d7a1bedb22adaf
-    if properties.connection.is_aws_msk_iam() {
-        #[cfg(not(madsim))]
-        client.poll(Duration::from_secs(10)); // note: this is a blocking call
-        #[cfg(madsim)]
-        client.poll(Duration::from_secs(10)).await;
-    }
     Ok(Arc::new(client))
```

Comment:
StreamConsumer only setup the polling interval at `max.poll.interval.ms`. We might need to keep this for initial polling

Reply:
But `StreamConsumer` does not expose the `poll` interface 🥵 https://docs.rs/rdkafka/latest/rdkafka/consumer/struct.StreamConsumer.html

Reply:
Yeah you did ask why using BaseConsumer in our discussion. Now I remember

Reply:
Let's see if it works.

Reply:
No 😒